src/Shopsys/ShopBundle/Resources/config/services.yml

```diff
@@ -42,3 +42,21 @@ services:
   Shopsys\FrameworkBundle\Model\Category\CategoryFactoryInterface: '@Shopsys\ShopBundle\Model\Category\CategoryFactory'
 
   Shopsys\FrameworkBundle\Model\Category\CategoryDataFactoryInterface: '@Shopsys\ShopBundle\Model\Category\CategoryDataFactory'
+
+  twig_cache_extension.cache_provider:
+    class: Doctrine\Common\Cache\RedisCache
+    factory: 'Shopsys\FrameworkBundle\Component\Doctrine\Cache\RedisCacheFactory:create'
+    arguments:
+      - '@snc_redis.twig'
+
+  twig_cache_extension.cache_strategy:
+    class: Shopsys\ShopBundle\Twig\CurrentDomainLifetimeCacheStrategy
+    arguments:
+      - '@twig_cache_extension.cache_provider'
+
+  twig_cache_extension:
+    class: Asm89\Twig\CacheExtension\Extension
+    arguments:
+      - '@twig_cache_extension.cache_strategy'
+    tags:
+      - { name: twig.extension }
```

Comment:
These tags are already covered by autoconfiguration: https://github.com/shopsys/demoshop/pull/9/files#diff-4314c4cdb737e9a7ed5c1dc455dabaafR7

It makes no sense to use them duplicitly

Reply:
Well it may have sense, could You please create an issue? ;-)

Reply:
Nono, it's just my code-review comment. You can handle it if it seems important